tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraFilterSetMethodContract.scala

```diff
@@ -399,7 +399,6 @@ trait LinagoraFilterSetMethodContract {
          |						"appendIn": {
          |							"mailboxIds": []
          |						},
-         |						"withKeywords": [],
```

Comment:
Q: Any issue/bug keeping this?

Reply:
No. We can have it or not. The results are the same

Reply:
I have updated code in james so that in case withKeywords field in json file does not exists, the empty list will be created for withKeywords in the Rule object